sdks/python/setup.py

```diff
@@ -215,6 +215,8 @@ def get_version():
     'google-cloud-language>=1.3.0,<2',
     'google-cloud-videointelligence>=1.8.0,<2',
     'google-cloud-vision>=0.38.0,<2',
+    # GCP packages required by fast container startup functionality.
```

Comment:
nit: "required by prebuild sdk container functionality". That is what you are calling it in options.